sdks/python/apache_beam/runners/direct/direct_runner.py

```diff
@@ -87,11 +87,7 @@ def run(self, pipeline):
                         evaluation_context)
     # Start the executor. This is a non-blocking call, it will start the
     # execution in background threads and return.
-
-    if pipeline.options:
-      # DirectRunner does not support RuntimeValueProviders.
-      RuntimeValueProvider.set_runtime_options(None, {})
-
+    RuntimeValueProvider.set_runtime_options({})
```

Comment:
Could you keep the comment `# DirectRunner does not support RuntimeValueProviders.`